cosmo_tester/test_suites/cluster/certificate_replacement_cluster_tests.py

```diff
@@ -7,7 +7,7 @@
 
 def test_replace_certificates_on_cluster(full_cluster_ips, logger, ssh_key,
                                          test_config, module_tmpdir):
-    broker1, broker2, broker3, db1, db2, db3, mgr1, mgr2 = full_cluster_ips
+    broker1, broker2, broker3, db1, db2, db3, mgr1, mgr2, _ = full_cluster_ips
```

Comment:
Probably best to put it as mgr3- we've got this in other files, I must've missed this one.